app/templates/edit_section.html

```diff
@@ -31,28 +31,19 @@
   %}
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
+
   {% if errors %}
-    <div class="validation-masthead">
-      <h2 class="validation-masthead-heading">
-        There were problems with your response to the following questions:
```

Comment:
Do we care about pluralisation here? The toolkit template uses the singular by default, so if we answer two questions wrong on the same page, the message is 

```
There was a problem with your answer to: 
Thing 1 
Thing 2
```

If we prefer the toolkit's singular version (I think it's probably better than pluralisation by default!), do we have any other pluralised error messages lurking around that need changing?

Reply:
I think the toolkit version is better (it's certainly newer).  As to whether there are other messages about... quite possibly.  I can have a look in this app for now.

Reply:
There are no obvious others anywhere in this repo.

Reply:
Would be nice if the toolkit version pluralised itself if appropriate.

Reply:
As long as it's consistent I think the toolkit version is fine for now (perhaps pluralisation is a nice 20% project for someone?).

Reply:
I think the singular version works just fine with a list of things under it?